README.md

```diff
@@ -1,3 +1,28 @@
-## About
+# About
 
-This is a (very) lightweight discord clone.
+Liscord is a (very) lightweight discord clone.
+
+# Requirements
+
+* Python 3.x
+
+# Installation
+
+* git clone https://github.com/raqibahmed/liscord
+
+```bash
+pip install -r requirements.txt
+```
+```bash
+pip install phonenumbers
```

Comment:
You can move this library to the `requirements.txt` file itself and reduce one step from the installation instructions.